core/payment-driver/src/dao/payment.rs

```diff
@@ -48,49 +48,49 @@ impl<'c> PaymentDao<'c> {
         .await
     }
 
-    pub async fn get_payment_status(&self, invoice_id: String) -> DbResult<Option<PaymentStatus>> {
-        //
-        readonly_transaction(self.pool, move |conn| {
-            let payment: PaymentEntity = match dsl::gnt_driver_payment
-                .find(&invoice_id)
-                .first(conn)
-                .optional()?
-            {
-                Some(v) => v,
-                None => return Ok(None),
-            };
-            let tx_id = payment.tx_id.clone();
-            let mut status = payment_entity_to_status(&payment);
-            if let PaymentStatus::Ok(ref mut confirmation) = &mut status {
-                let tx_id = match tx_id {
-                    Some(v) => v,
-                    None => {
-                        log::error!("invalid payment state (invoice={})", invoice_id);
-                        return Ok(Some(PaymentStatus::Unknown));
-                    }
-                };
-
-                let tx: TransactionEntity =
-                    tx_dsl::gnt_driver_transaction.find(&tx_id).first(conn)?;
-                if tx.status != TX_CONFIRMED {
-                    return Ok(Some(PaymentStatus::NotYet));
-                }
-                let tx_hash = match tx.tx_hash {
-                    Some(h) => hex::decode(h).map_err(|e| DbError::InvalidData(e.to_string()))?,
-                    None => {
-                        log::error!("invalid payment state (invoice={})", invoice_id);
-                        return Ok(Some(PaymentStatus::Unknown));
-                    }
-                };
-                *confirmation = PaymentConfirmation {
-                    confirmation: tx_hash,
-                };
-            }
-
-            Ok(Some(status))
-        })
-        .await
-    }
+    // pub async fn get_payment_status(&self, invoice_id: String) -> DbResult<Option<PaymentStatus>> {
```

Comment:
@bidzyyys these break my compilation, please bring them back ( or delete them ) when updating the GNT driver

Reply:
yes, when these functions are commented out it compiles :)

Thank you <3